integration_tests/src/utils.rs

```diff
@@ -261,3 +324,370 @@ pub fn set_sigterm() {
     })
     .expect("Error setting Ctrl-C handler");
 }
+
+/// Run an iperf to generate from between two namespaces
+pub fn generate_traffic(from: Namespace, to: Namespace) {
+    let ip = format!("fd00::{}", to.id);
+    // setup server
+    info!("Going to setup server, spawning new thread");
+    thread::spawn(move || {
+        let _output = KI
+            .run_command("ip", &["netns", "exec", &to.get_name(), "iperf3", "-s"])
+            .expect("Could not setup iperf server");
+    });
+
+    // iperf client
+    info!("Going to setup client");
+    let output = KI
+        .run_command(
+            "ip",
+            &["netns", "exec", &from.get_name(), "iperf3", "-c", &ip],
+        )
+        .expect("Could not setup iperf client");
+    let output = from_utf8(&output.stdout).expect("could not get output for client setup!");
+    info!("Client out: {}", format!("{}", output));
+}
+
+pub async fn query_debts(node: Namespace, for_node: Namespace) -> GetDebtsResult {
+    let for_node_ip = match for_node.id {
+        4 => EXIT_ROOT_IP.to_string(),
```

Comment:
we should check if the namespace is an exit, rather than relying on hardcoding id's

one of the next tests we want to implement is a multi exit test to test exit roaming

Reply:
Done